core/modules/wdb/QueryRunner.h

```diff
@@ -121,6 +121,8 @@ class QueryRunner : public wbase::TaskQueryRunner, public std::enable_shared_fro
 
     wbase::Task::Ptr const _task; ///< Actual task
 
+    size_t const _maxResultSize = 5500000000; ///< &&& remove ???
```

Comment:
This limit has to be loaded from a configuration of "czar". If you don't want to set any specific limit (I may see scenarios when that could be the case) then I'd rather initialize this attribute with 0 (or with `std::numeric_limits<size_t>::max()`. Then you would have to check in the rest of your code if it's set to 0 and don't enforce the limit. Just post a warning in the Logger stream.

Reply:
Deleted. I had thought I'd already gotten rid of all of this, but I missed some.